citizen/citizen.c

```diff
@@ -327,8 +366,9 @@ Citizen *searchCitizenById(List *list, int citizenId) {
 }
 
 
-//Get the age of the citizen
-int getCitizenAge(List *list, int citizenId) {
+// Get the age of the citizen
+int getCitizenAge(List *list, int citizenId)
```

Comment:
please follow the naming conventions on the variables, use snake_case

Reply:
Applied, thanks.